backend/config/curation-api.yml

```diff
@@ -780,6 +780,7 @@ components:
         - COUNT
         - NORMAL
       type: string
+      nullable: true
```

Comment:
I could only find `x_approximate_distribution`, not `distribution`, in `DbDataset` (am I looking in the right spot?)